test_scripts/API/SetAudioStreamingIndicator/034_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_true.lua

```diff
@@ -0,0 +1,105 @@
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+-- HMI->SDL: UI.SetAudioStreamingIndicator(resultcode: HMI_result_code)
+--
+-- Expected result:
+-- SDL->HMI: UI.SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+-- SDL->mobile: SetAudioStreamingIndicator_response(HMI_result_code, success:true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+-- in scope of the CRQ info parameter is not specified, but will be left for any future use.
+local hmi_result_code = {
+	{ result_code = "SUCCESS", info = "" },
+	{ result_code = "WARNINGS", info = "" },
+	{ result_code = "WRONG_LANGUAGE", info = "" },
+	{ result_code = "RETRY", info = "" },
+	{ result_code = "SAVED", info = "" },
+	{ result_code = "UNSUPPORTED_RESOURCE", info = "" },
+}
+
+--[[ General Precondition before ATF start ]]
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"}, "SetAudioStreamingIndicator")
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+for i = 1, #hmi_result_code do
+	Test["TestStep_SetAudioStreamingIndicator_"..hmi_result_code[i].result_code.."_audioStreamingIndicator_PAUSE"] = function(self)
+	  local corr_id = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+
+	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+	  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, data.method, hmi_result_code[i].result_code) end)
+
+	  EXPECT_RESPONSE(corr_id, { success = true, resultCode = hmi_result_code[i].result_code})
+	  EXPECT_NOTIFICATION("OnHashChange",{}):Times(0)
+	end
+
+	function Test:Postcondition_SetAudioStreamingIndicator_SUCCESS_audioStreamingIndicator_PLAY()
```

Comment:
@istoimenova As I understand Iliyana "reset" audioStreamingIndicator value to avoid IGNORED resultCode on second request wiht "PAUSE.
Iliyana what about changing audioStreamingIndicator value depends of example "i" parity? 
Some like:

> local IndicatorValue = "PAUSE"
> if math.fmod(i,2)~=0 then
>   IndicatorValue = "PAUSE"
> else 
>   IndicatorValue = "PLAY"
> end